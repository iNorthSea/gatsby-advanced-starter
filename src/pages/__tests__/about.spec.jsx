import React from "react";
import { render } from "@testing-library/react";
import AboutPage from "../about";

describe("page component AboutPage", () => {
  it("renders correctly", () => {
    expect.assertions(1);
    const { asFragment } = render(<AboutPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
