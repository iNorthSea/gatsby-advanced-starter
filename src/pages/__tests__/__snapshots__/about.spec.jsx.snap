// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page component AboutPage renders correctly 1`] = `
<DocumentFragment>
  <div
    class="layout-container"
  >
    <div
      class="about-container"
    >
      <div
        class="about"
      >
        <h1>
          Edit About component or pages/about.jsx to include your information.
        </h1>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
